crates/bridge-sig-manager/src/operations.rs

```diff
@@ -113,6 +113,14 @@ pub fn sign_state_partial(
 }
 
 /// Verify that a partial MuSig2 signature is correct.
+///
+/// WARNING: It is possible for an adverary to forge a partial signature under certain conditions.
```

Comment:
Could you link to any relevant resources for what these conditions are? This is both for this documentation and for myself. 😅

Reply:
Sure, I'll add that. It's in a footnote of section 4.2 in [the preprint](https://eprint.iacr.org/2020/1261), which links out [here](https://gist.github.com/AdamISZ/ca974ed67889cedc738c4a1f65ff620b).

Reply:
I also need to change the statement about failed partial verification guaranteeing a failed signing session. It's possible for an adversary to manipulate multiple (otherwise valid) partial signatures such that each will fail, but the signing session will not. It's something of a weird edge case.

Reply:
Maybe we should move these to H1? The same way we suggest here: https://www.notion.so/Engineering-Rust-Best-Practices-14c901ba000f8025a974ec8e2a70b4af?pvs=4#14d901ba000f8020aebbc58bee7c0a4c

as in:

```rust
/// Foo bla bla bla
///
/// # Warning
///
/// It is possible for an adversary to forge
```

And how did `codespell` not got this `adverary`?